tests/src/TwitchPubSubClient.cpp

```diff
@@ -107,38 +107,40 @@ TEST(TwitchPubSubClient, ServerRespondsToPings)
     MockApplication a("", 1s);
     auto &pubSub = a.pubSub;
 
-    pubSub.start();
+    QTest::qWait(50);
 
-    std::this_thread::sleep_for(50ms);
-
-    ASSERT_EQ(pubSub.diag.connectionsOpened, 0);
-    ASSERT_EQ(pubSub.diag.connectionsClosed, 0);
-    ASSERT_EQ(pubSub.diag.connectionsFailed, 0);
+    ASSERT_EQ(pubSub.wsDiag().connectionsOpened, 0);
+    ASSERT_EQ(pubSub.wsDiag().connectionsClosed, 0);
+    ASSERT_EQ(pubSub.wsDiag().connectionsFailed, 0);
     ASSERT_EQ(pubSub.diag.messagesReceived, 0);
 
     pubSub.listenToChannelPointRewards("123456");
 
-    std::this_thread::sleep_for(150ms);
+    QTest::qWait(150);
 
-    ASSERT_EQ(pubSub.diag.connectionsOpened, 1);
-    ASSERT_EQ(pubSub.diag.connectionsClosed, 0);
-    ASSERT_EQ(pubSub.diag.connectionsFailed, 0);
-    ASSERT_EQ(pubSub.diag.messagesReceived, 2);
+    ASSERT_EQ(pubSub.wsDiag().connectionsOpened, 1);
+    ASSERT_EQ(pubSub.wsDiag().connectionsClosed, 0);
+    ASSERT_EQ(pubSub.wsDiag().connectionsFailed, 0);
+    ASSERT_EQ(pubSub.diag.messagesReceived, 1);  // LISTEN
```

Comment:
what changed here to make this 1 instead of 2?

Reply:
Previously, pings were sent out immediately. Now, we wait for the heartbeat/ping timer to expire once.